backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/impl/ShipmentSchedulePA.java

```diff
@@ -433,6 +445,18 @@ public void updatePreparationDate_Override(final Timestamp preparationDate, fina
 		);
 	}
 
+	@Override
+	public void updateExportStatus(final String exportStatus, final PInstanceId pinstanceId)
+	{
+		updateColumnForSelection(
+				I_M_ShipmentSchedule.COLUMNNAME_ExportStatus,
+				exportStatus,
+				false,
```

Comment:
```suggestion
				false /* updateOnlyIfNull */,
```
most IDEs & Editors don't display the parameter name